### PR TITLE
[Profile] Add Art's Signal trading

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

File: profiles/arts_signal_scalp_trading/profile.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            }
+        },
+        "trading": {
+            "reference-market": "USDT",
+            "risk": 1
+        }
+    },
+    "profile": {
+        "avatar": "default_profile.png",
+        "description": "Arts Signal Scalp trading connects OctoBot to Art's GodBot scalp telegram signals.",
```
Comment (GuillaumeDSM) [inline profiles/arts_signal_scalp_trading/profile.json:38]: Should we include the telegram group url ?

Comment (Herklos) [reply profiles/arts_signal_scalp_trading/profile.json:38]: Yes :+1:

File: Evaluator/Social/signal_evaluator/config/TelegramChannelSignalEvaluator.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "channels": {
-    "WallStreetBets Events 🛸": {
-      "signal_pattern": "The coin we have picked to pump today is :  #(.*)",
```
Comment (GuillaumeDSM) [inline Evaluator/Social/signal_evaluator/config/TelegramChannelSignalEvaluator.json:4]: 😂

Comment (Herklos) [reply Evaluator/Social/signal_evaluator/config/TelegramChannelSignalEvaluator.json:4]: An old experience... :smile:

File: Evaluator/Social/signal_evaluator/signal.py [hunk 3]
```diff
@@ -121,12 +120,12 @@ async def _feed_callback(self, data):
...
-                if signal is not None:
-                    self.eval_note = -1
-                    await self.evaluation_completed(signal,
-                                                    symbol_util.merge_currencies(signal,
-                                                                                 channel_data[self.SIGNAL_MARKET_KEY]),
-                                                    eval_time=self.get_current_exchange_time())
+                pair = self._get_signal_message(channel_data[self.SIGNAL_PAIR_KEY], message)
+                if signal is not None and pair is not None:
+                    self.eval_note = int(signal)
+                    await self.evaluation_completed(cryptocurrency="BTC", eval_time=self.get_current_exchange_time())
```
Comment (GuillaumeDSM) [inline Evaluator/Social/signal_evaluator/signal.py:126]: "BTC" ? ^^

File: Evaluator/Social/signal_evaluator/config/TelegramChannelSignalEvaluator_schema.json [hunk 4]
```diff
@@ -1,19 +1,31 @@
...
-            },
-            "signal_market" : {
-                "type" : "string"
-            }
+        "properties": {
+          "MARKET_BUY": {
+            "type": "string"
+          },
+          "MARKET_SELL": {
+            "type": "string"
```
Comment (GuillaumeDSM) [inline Evaluator/Social/signal_evaluator/config/TelegramChannelSignalEvaluator_schema.json:22]: should we add placeholders or descriptions ?

Comment (Herklos) [reply Evaluator/Social/signal_evaluator/config/TelegramChannelSignalEvaluator_schema.json:22]: We can yes

Comment (Herklos) [reply Evaluator/Social/signal_evaluator/config/TelegramChannelSignalEvaluator_schema.json]: I added titles but I don't think adding placeholders is relevant.

**Review by GuillaumeDSM — APPROVED** [review]
🎉

**Pull request merged**